src/components/home/Home.vue

```diff
@@ -44,18 +37,21 @@ export default {
     Modal,
     UserInfoForm,
     Form,
+    ActivityButtons,
   },
   setup() {
     const FFMI = ref(0);
     const lowTempo = ref(0.3);
     const fastTempo = ref(0.9);
     const isModalOpened = ref(false);
+    const activityOption = ref({ name: null, kcal: null });
 
     return {
       FFMI,
       lowTempo,
       fastTempo,
       isModalOpened,
+      activityOption,
       openModal: () => (isModalOpened.value = true),
       closeModal: () => (isModalOpened.value = false),
```

Comment:
może coś w stylu `toggleModal: () => isModalOpened.value = !isModalOpened.value` by tutaj mogło sensownie działać? tylko sugestia, pozostawiam to do Twojej oceny ^^

Reply:
Ten zapis jest krótszy, ale może póki jeszcze nie jesteśmy tacy zaawansowani, a dajemy komuś do przejrzenia kod to może by to dla niego czytelniejsze